client/my-sites/email/email-providers-stacked-comparison/index.tsx

```diff
@@ -94,6 +105,17 @@ const EmailProvidersStackedComparison: FunctionComponent< EmailProvidersStackedC
 				{ translate( 'Pick an email solution' ) }
 			</h1>
 
+			<div className="email-providers-stacked-comparison__how-they-compare">
+				{ translate( 'Not sure how to start?' ) }
```

Comment:
I think it would be better for translations to have all the text in the same string, as it gives more context to translators:

```suggestion
				{ translate( 'Not sure how to start? {{a}}See how they compare{{/a}}.' ) }
```